pkg/componentarchive/archive.go

```diff
@@ -89,10 +91,35 @@ func (o *BuilderOptions) Build(fs vfs.FileSystem) (*ctf.ComponentArchive, error)
 			if err != nil {
 				return nil, fmt.Errorf("unable to create projectionfilesystem: %w", err)
 			}
-			archive, err := ctf.NewComponentArchiveFromFilesystem(archiveFs)
+
+			decodeOpts := []codec.DecodeOption{
+				codec.DisableValidation(true),
+			}
+			archive, err := ctf.NewComponentArchiveFromFilesystem(archiveFs, decodeOpts...)
 			if err != nil {
 				return nil, fmt.Errorf("unable to parse component archive from %s: %w", o.ComponentArchivePath, err)
 			}
+
+			cd := archive.ComponentDescriptor
+
+			if o.Name != "" {
+				if cd.Name != "" && cd.Name != o.Name {
```

Comment:
```suggestion
				if cd.Name != o.Name {
```

Reply:
the suggested change doesn't work for cases where the existing cd name/version is empty and should be overwritten

Reply:
hm but the statement `cd.Name != ""` is always true so you could also write `if true && cd.Name != o.Name {`.
Or am I missing something?

Reply:
no cd.Name can be empty. I disabled the validation when loading the cd from the hard drive exactly for this case, which was the initial problem of the bug. and in case the name/version is empty, overwriting should be allowed.

Reply:
ah yes you're sorry, miss-read the line above👍🏻

Reply:
then everything is fine